solr/core/src/java/org/apache/solr/core/backup/repository/LocalFileSystemRepository.java

```diff
@@ -57,6 +57,9 @@ public void init(NamedList<?> args) {
     baseLocation =
         Optional.<String>ofNullable(getConfigProperty(CoreAdminParams.BACKUP_LOCATION))
             .map(Path::of)
+            .or(
+                () -> Optional.ofNullable(System.getProperty("solr.backups.path")).map(Path::of)
```

Comment:
Is it not possible to have multiple backup repos defined at the same time? Thus, this prop should probably qualify which one, e.g. `solr.backups.localfs.path`?

Reply:
So I think the LocalFileSystemRepository is the only one that is implicitly created. (All others have to be explicitly mentioned in the solr.xml). That's why this would be necessary in the first place...